l10n/de_DE.js

```diff
@@ -70,6 +70,10 @@ OC.L10N.register(
     "Voice" : "Anruf",
     "Social network" : "Soziales Netzwerk",
     "Details" : "Details",
-    "Settings" : "Einstellungen"
+    "Settings" : "Einstellungen",
+    "Gender" : "Geschlecht",
+    "Male" : "männlich",
+    "Female" : "weiblich",
+    "intersex" : "intersexuell"
```

Comment:
@hirschrobert That's correct - you just have to add "t('contacts', ...)" in the code and the rest is handled automatically.